x-pack/plugins/observability_solution/logs_data_access/common/constants.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export const DEFAULT_LOG_SOURCES = ['logs-*-*'];
+export const DEFAULT_LOG_SOURCES = ['logs-*,filebeat-*,kibana_sample_data_logs*'];
```

Comment:
I assume you removed `logs-*-*` from the list as `logs-*` already covers it. I would like to keep it in the list as I think it is the one that users should eventually migrate to, not `logs-*`. I would keep both.

Reply:
Yep. Okay, no problem, I'll add it back in 👍